src/app/home/navigation.component.ts

```diff
@@ -9,7 +9,7 @@ import { Router } from '@angular/router';
     <ul>
       <li *ngFor="let comp of components"><a [routerLink]="'/' + comp.link">{{comp.name.toUpperCase()}}</a></li>
       <li><a href="#" class="km-logout" (click)="logoutClick()">LOGOUT</a></li>
-      <li><a routerLink="/manager"><i class="fa fa-user" aria-hidden="true"></i></a></li>
+      <li><a routerLink="/manager"><mat-icon>person</mat-icon></a></li>

```

Comment:
Nice but can u resize icon as it has effected in navigation-bar or else there might be material  icon like medium..

Reply:
this PR only deals with removing bootstrap, popper and fontawesome and repllacing fa icon with material icon not ablout styling or designing

Reply:
then why did u change code right here navigation.component.ts as it results :
![see](https://user-images.githubusercontent.com/24644003/33652464-aa33d984-da91-11e7-8810-2fb8eac20449.png)

Reply:
as far as i was told to keep everything as it is and not to touch designing par or styling part